css/login.css

```diff
@@ -0,0 +1,55 @@
+/*Color and shades not finalised yet, will use color variables later*/
+.login-wrapper {
+  background-color: aliceblue;
+  margin: auto;
+  margin-top: 25vh;
```

Comment:
Instead of adding a `margin: 25vh` use `flex` to center it on the screen.

Reply:
I don't want the page to be centered for now, as I will be adding some footer and make the login page slightly left. Will make these changes then. Thanks for suggesting this.